src/main/java/com/amazonaws/kinesisvideo/demoapp/activity/WebRtcActivity.java

```diff
@@ -259,6 +265,39 @@ private boolean isValidClient() {
         return client != null && client.isOpen();
     }
 
+    private void handlePendingIceCandidates(String clientId) {
+        // Check if ICE candidates are available in the queue and add the candidate
+        while(!pendingIceCandidatesQueue.isEmpty()) {
+            final Event message = pendingIceCandidatesQueue.peek();
+            final IceCandidate iceCandidate = Event.parseIceCandidate(message);
+            if(clientId == message.getSenderClientId()) {
```

Comment:
equals?

Reply:
My bad. Missed that it is of string data type. Fixed it.